src/Pos/Wallet/Web/Api.hs

```diff
@@ -250,6 +255,7 @@ instance ToSchema      SyncProgress
 instance ToSchema      ChainDifficulty
 instance ToSchema      BlockVersion
 instance ToSchema      BackupPhrase
+instance ToParamSchema CPassPhrase
 
 -- | Build Swagger-specification from 'walletApi'.
 swaggerSpecForWalletApi :: Swagger
```

Comment:
I guess, It was in master when I branched out, later it was removed. This comment points to old commit, this stuff doesn't exist in the PR anymore